WAITLIST.s

```diff
@@ -503,7 +503,7 @@ LNGCALL2	LXCH	LONGEXIT +1	# SAVE THE CORRECT BB FOR RETURN
 
 # *** WAITLIST TASK LONGCYCL ***
 
-LONGCYCL	EXTEND			# CAN WE SUCCESFULLY TAKE ABOUT 1.25
+LONGCYCL	EXTEND			# CAN WE SUCCESSFULLY TAKE ABOUT 1.25
```

Comment:
`SUCCESFULLY` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1234.jpg)